packages/mco/components/create-dr-policy/select-replication-type.tsx

```diff
@@ -29,6 +32,20 @@ export const normalizeSyncTimeValue = (value: number) => {
   return syncTimeValue < MIN_VALUE ? MIN_VALUE : syncTimeValue;
 };
 
+const checkSyncPolicyAlreadyExists = (
+  drPolicies: DRPolicyKind[],
+  selectedClusters: string[]
+): boolean =>
+  drPolicies.some((drPolicy) => {
+    const { drClusters, schedulingInterval } = drPolicy.spec;
+    const isSyncPolicy =
+      getReplicationType(schedulingInterval) === REPLICATION_TYPE.SYNC;
+    const isClustersMatches = drClusters.every((cluster) =>
+      selectedClusters.includes(cluster)
+    );
+    return isSyncPolicy && isClustersMatches;
```

Comment:
nits (optional): no need to run `every` if policy is `async`...
```suggestion
    return isSyncPolicy && drClusters.every((cluster) =>
      selectedClusters.includes(cluster)
    );
```